hobbify_stats/postgres.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+"""This script contains the connection with postgres"""
+import os
+import psycopg2
+
+POSTGRES_HOST = os.environ['POSTGRES_HOST']
+POSTGRES_PORT = os.environ['POSTGRES_PORT']
+POSTGRES_DB = os.environ['POSTGRES_DB']
+POSTGRES_USER = os.environ['POSTGRES_USER']
+POSTGRES_PASSWORD = os.environ['POSTGRES_PASSWORD']
```

Comment:
Use `os.getenv()` instead. This can yield KeyError.

[Docs](https://docs.python.org/3.8/library/os.html#os.getenv).